patches/gulp-livereload++chalk+2.4.2.patch

```diff
@@ -0,0 +1,47 @@
+diff --git a/node_modules/gulp-livereload/node_modules/chalk/index.js b/node_modules/gulp-livereload/node_modules/chalk/index.js
+index 1cc5fa8..08a1f17 100644
+--- a/node_modules/gulp-livereload/node_modules/chalk/index.js
++++ b/node_modules/gulp-livereload/node_modules/chalk/index.js
+@@ -47,14 +47,7 @@ function Chalk(options) {
+ 	applyOptions(this, options);
+ }
+ 
+-// Use bright blue on Windows as the normal blue color is illegible
```

Comment:
The build seems to work without this patch. This package is used for livereload, so I tested this by running `yarn start` and changing files, to see whether anything broke.

Reply:
this one can be removed. latest LavaMoat no longer protects module exports : (

Reply:
this patch should be removed